opentelemetry/proto/profiles/v1development/profiles.proto

```diff
@@ -407,10 +407,10 @@ message Sample {
 // Provides additional context for a sample,
 // such as thread ID or allocation size, with optional units. [deprecated]
 message Label {
```

Comment:
Should we get rid of the Label type? Keeping the use of both attributes and labels was part of trying to be wire compatible with pprof but we don't pursue that lower level goal anymore.

Reply:
Not in this PR, mostly want to discuss going forward.